### PR TITLE
Embed stream end to end testing

### PR DESCRIPTION
**Pull request by davidgljay**
## What does this PR do?

Adds end to end testing to the embed stream.

## How do I test this PR?

1. `npm run e2e`
2. Tests should run and pass.

## Notes

- There is one tests missing from spec here: https://gist.github.com/kgardnr/b225638ddaa13de8e90f0f717e4e220b
  - This timestamp test is better handled via unit testing. It doesn't rely on the integrated system and requires that we fake the timestamp of a comment (which is something that our system should generally discourage anyone from doing.)

@coralproject/tech

### CODE REVIEW

**Comment by davidgljay** [comment]
I've created an initial PR with e2e tests, but would love help completing over the holidays. @okbel, would you be ok wrapping this up?

**Comment by davidgljay** [comment]
This PR is not ready to be merged.

**Review by wyattjoh — DISMISSED** [review]
I have changes pending that are required before this can merge.

**Comment by davidgljay** [comment]
@wyattjoh, Do you need me to hold this branch or is it ok to merge in?

**Comment by davidgljay** [comment]
Addressed issue in the way that the mongo db path was being set.

File: .env-test
Comment (wyattjoh) [inline .env-test]: We should remove this file b4 merge.

**Comment by okbel** [comment]
@wyattjoh Can we merge this PR? /cc @davidgljay

**Review by okbel — APPROVED** [review]

**Review by okbel — APPROVED** [review]

**Pull request merged**
